src/pages/blog.js

```diff
@@ -4,18 +4,16 @@ import Layout from '../components/layout'
 import Byline from '../components/common/byline'
 
 export default ({ data }) => (
-  <Layout title="Blog" textHeavy>
+  <Layout title="Blog" textHeavy narrow>
     {data.allContentfulBlogPost.edges.map(({ node }) => (
       <>
-        <h3>
+        <h2>
           <Link to={`/blog/${node.slug}`}>{node.title}</Link>
-        </h3>
+        </h2>
         <Byline author={node.author} date={node.publishDate} />
         <div
           dangerouslySetInnerHTML={{
-            __html:
-              node.childContentfulBlogPostTeaserTextNode.childMarkdownRemark
-                .html,
+            __html: node.lede,
```

Comment:
Since lede is regular text, let's get rid of this div and just use:

`<p>{node.lede}</p>`

Reply:
Good call, yep. I'm adding the same `lede` class as we have in the individual post pages, too.